screwdriver.yaml

```diff
@@ -1,5 +1,5 @@
 shared:
-    image: golang:1.10
+    image: golang
```

Comment:
Does this default to "latest", and is that safe?

Reply:
i think this is fine - Go is very stable in terms of backward compatibility https://golang.org/doc/go1compat.